rpc-rs/tests/nats_sub.rs

```diff
@@ -136,12 +133,12 @@ async fn listen_queue(
             }
             count += 1;
         }
-        println!("subscriber '{}' exiting count={}", &subject, count);
+        info!("subscriber '{}' exiting count={}", &subject, count);
         count
     })
 }
 
-#[tokio::test]
+#[test(tokio::test(flavor = "multi_thread"))]
```

Comment:
I've never seen these wrapped in `test` before. There a reason this is needed?

Reply:
This is part of test_log crate that initializes tracing/logging for tests